DESCRIPTION

```diff
@@ -7,7 +7,7 @@ Description: The Crunch.io service <https://crunch.io/> provides a cloud-based
     datasets from within R. Importantly, this allows technical researchers to
     collaborate naturally with team members, managers, and clients who prefer a
     point-and-click interface.
-Version: 1.31.0
+Version: 1.31.0.9000
```

Comment:
Going with tidyverse conventions on dev versions - by having a `9000` it's clear it's a dev version, but I still have room to increment by 1 when I make breaking changes.

(YG had issues since I was making changes without incrementing versions)